Palaso/Resources/CountryCodes.txt

```diff
@@ -44,13 +48,16 @@ CL	Chile	Americas
 CM	Cameroon	Africa
 CN	China	Asia
 CO	Colombia	Americas
+CP	Clipperton Island
 CR	Costa Rica	Americas
 CU	Cuba	Americas
 CV	Cape Verde Islands	Africa
 CW	Curacao	Americas
 CY	Cyprus	Asia
+CX	Christmas Island	Pacific
```

Comment:
Just noticed that this file is sorted, so this line should probably come before CY.